torch/_inductor/codegen/cpp.py

```diff
@@ -1665,23 +1711,64 @@ def gen_kernel(kernel):
                 if hasattr(kernel, "codegen_inner_loops"):
                     code.splice(kernel.poststores)
 
+            def replace_parallel_reduction_store(loops):
```

Comment:
```suggestion
            def update_stores_with_parallel_reduction(loops):
```

Reply:
Can we make it a kernel method since the variable names are defined by kernels.

Reply:
Can we move the update before https://github.com/pytorch/pytorch/pull/115620/files#diff-5ab7b0235e2076a5fc6629ba0b109208940f5b94f5c13babc3e0f87cf4fcec82R1803?